_layouts/ontology_detail.html

```diff
@@ -106,16 +106,48 @@ <h2>Products</h2>
             </tbody>
           </table>
         </div>
-        
+        {% if page.usages %}
+      <div>
+          <h2> Usages </h2> <ul>
+        {% for p in page.usages %}
+          <li>
+           <b> User: </b> <a href="{{p.user}}"> {{p.user}} </a> <br/>
```

Comment:
Please don't use `<br/>`, since it has no semantics. Try using `<did>` and `<dt>`, as in the sidebar.